src/sqlfluff/core/templaters/jinja.py

```diff
@@ -354,15 +354,20 @@ def _slice_template(cls, in_str: str) -> Iterator[RawFileSlice]:
                         trimmed_content = str_buff[
                             len(m_open.group(0)) : -len(m_close.group(0))
                         ]
+                    first_word = trimmed_content.split()[0]
                     if trimmed_content.startswith("end"):
                         block_type = "block_end"
                     elif trimmed_content.startswith("el"):
                         # else, elif
                         block_type = "block_mid"
-                    else:
+                    elif first_word in ("for", "if"):
                         block_type = "block_start"
-                        if trimmed_content.split()[0] == "for":
+                        if first_word == "for":
                             block_subtype = "loop"
+                    else:
+                        # Not necessarily a comment, but we can treat it as one
+                        # for our purposes.
+                        block_type = "comment"
```

Comment:
This change fixes an indentation issue -- a line like this:
```
{% set considered_actions=['a1', 'a2', 'a3', 'a4', 'a5'] %}
```

was previously labeled as `block_type` = `block_start`. That caused other parts of the code to be over-indented.